spec/rails_helper.rb

```diff
@@ -0,0 +1,100 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+require 'capybara/rspec'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
:bulb: [OPTIONAL] These two lines of `require` statements **caused issues** on my side. I had to delete tom in order to run the tests. Consider deleting them, or please instruct users about how to run tests 🙏